layout/_partials/comments.swig

```diff
@@ -64,7 +64,6 @@
 
   {% elseif theme.valine.appid and theme.valine.appkey %}
     <div class="comments" id="comments">
-      <div id="vcomments"></div>
```

Comment:
No need apeear comments block on full post here?

Reply:
Ok, i got it. thx